.gitignore

```diff
@@ -50,3 +50,11 @@ yarn-debug.log*
 .DS_Store
 # This is ignored due to Ansible I assume:
 config/initializers/comfortable_mexican_sofa.rb
+
+# Config Gem: local dev overrides
+config/settings.local.yml
+config/settings/*.local.yml
+config/environments/*.local.yml
```

Comment:
Added by `rails g` and allows overriding config with local config ignored by Git